hack/generator/pkg/astmodel/map_type.go

```diff
@@ -52,6 +53,11 @@ func (m *MapType) AsType(codeGenerationContext *CodeGenerationContext) dst.Expr
 	}
 }
 
+// AsZero renders an expression for the "zero" value of a map by calling make()
+func (m *MapType) AsZero(_ Types, ctx *CodeGenerationContext) dst.Expr {
+	return astbuilder.CallFunc("make", m.AsType(ctx))
```

Comment:
The zero value of a map should be nil, rather than an empty map.

Reply:
Fixed.